cdk_integration_tests/utils.py

```diff
@@ -14,12 +14,18 @@ def load_failed_checks_from_file(lang: str) -> Dict[str, List[Dict[str, Any]]]:
         assert report is not None
         results = report.get("results", {})
         failed_checks = results.get("failed_checks")
+        skipped_checks = results.get("skipped_checks")
         results = {}
         for check in failed_checks:
             check_id = check['check_id']
             if not results.get(check_id):
                 results[check_id] = []
             results[check_id].append(check)
+        for check in skipped_checks:
```

Comment:
Is it should be here? it's skipped not failed...

Reply:
In prisma sometime the aws checks are suppressed
We dont want to break the pipeline because of it